lib/elastic_apm/graphql.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module ElasticAPM
+  # @api private
+  module GraphQL
+    TYPE = 'app'
+    SUBTYPE = 'graphql'
+
+    PREFIX = 'GraphQL: '
+    UNNAMED = '[unnamed]'
+    MORE_THAN_FIVE = '[more-than-five-queries]'
```

Comment:
It seems a bit strange to hardcode the query limit in the transaction name. I looked at the spec in the apm repo and the limit 5 seems to be arbitrary. What if that number changes? Would it be worth changing this to "exceeds-max-number-of-queries" in case that is configurable at some point? Or is it unlikely that number will change?

Reply:
That's a good point 👍